site/docs/types.md

```diff
@@ -29,13 +29,25 @@ Reference documentation for common parameter **types** expected by Vega-Lite spe
 
 Accepts any literal value, including a string, number, boolean, or `null`.
 
+<a name="align" href="#align">#</a>
+**Align**
```

Comment:
I don't think string enum like Align should be a special type in the docs. Feel like an overkill to jump to another page for simple string enum.

Reply:
Done